src/main/java/com/conveyal/datatools/manager/controllers/api/DeploymentController.java

```diff
@@ -95,11 +93,10 @@ private static Deployment deleteDeployment (Request req, Response res) {
     /**
```

Comment:
On [Line 62](https://github.com/ibi-group/datatools-server/pull/332/files#diff-cd2ed281b92f7a722ca238d2ae971be7R62), as @br648 pointed out on Teams, an EC2 client is built not using AWSUtils. This should be changed, right?